src/main.ts

```diff
@@ -29,6 +30,11 @@ Vue.use(Affix)
 Vue.use(Vue2Filters) // needed by SbcFeeSummary
 
 const store = getVuexStore()
+const pinia = getPiniaStore()
+
+// Needed to fix "getActivePinia was called with no active Pinia" error
+// Type assertion to turn off type checking
+Vue.use(pinia as any)
```

Comment:
Tell me more about the error.

Reply:
So, I spent a good amount of time on this.

For Pinia, if you try to access the store before having an active Pinia instance, you will get an error. Now what makes Filings UI different is that we use the `loadConfiguration()` method from the `configurationStore` in `main.ts`

At that instance, we don't have an active instance with Vue. So, in order to fix that in Vue 2, I had to use `Vue.use(pinia). Now, if I don't type assert as any, typescript was complaining. 

Ref: https://stackoverflow.com/questions/74865884/cant-access-pinia-store-in-beforeenter-in-vue-2-app

Reply:
Oh, OK. Please just add that reference to the code comment, and I'm OK with this.

Reply:
Thanks.